extending-valtimo/access-control/create-relation-between-resources.md

```diff
@@ -0,0 +1,96 @@
+# Creating a relation to another resource
+
+When setting conditions for a permission you sometimes want those conditions to apply to related resources. For example,
+you might want to view tasks for cases that have not yet been assigned. In this case the not assigned part do not apply
+directly to the task, but to the related case resource. To define conditions on related resources there first has to be 
+a relation that Valtimo can use to base its logic on. This page describes how to create a relation from one resource 
+to another.
+
+## How to create a mapper
+
+In order to support conditions on a related entity an implementation of `AuthorizationEntityMapper` needs to be created.
+This will tell Valtimo how to access the related data. Two different ways of accessing the data are supported for two
+scenarios.
+
+- When requesting a list of the primary resource with a condition on the related resource, the SQL query will have to be
```

Comment:
While this reads in a more natural way, I think it would beneficial to start with the method before describing the scenario. Just to make it easier for people to find what they are looking for. What do you think?